src/wemmick/utils.py

```diff
@@ -14,3 +15,36 @@ def file_relative_path(dunderfile, relative_path: str):
     H/T https://github.com/dagster-io/dagster/blob/8a250e9619a49e8bff8e9aa7435df89c2d2ea039/python_modules/dagster/dagster/utils/__init__.py#L34
     """
     return os.path.join(os.path.dirname(dunderfile), relative_path)
+
+
+def get_scheme(path: str):
+    return urlparse(path).scheme
+
+
+def check_file_exists(path):
+    if not os.path.isfile(path):
+        raise ValueError(
+            f"File {path} was not found. Please check the path and try again."
+        )
+
+
+def get_file(path: str):
+    scheme = get_scheme(path)
+    if scheme == "s3":
```

Comment:
Can python do a map and reducer like in js instead of the elif ? (just currious)

Reply:
Will look more into this

Reply:
So far I found [this pattern](https://hackernoon.com/how-to-properly-use-switch-case-statements-in-python-explained-with-examples-ivg536vx). Will consider it when adding S3 and GCS options.

Reply:
Nice, that was a great reference and helped my understanding python a bit more.  I particularly like the dict mapping to functions.